code/modules/mob/living/simple_animal/abnormality/aleph/titania.dm

```diff
@@ -150,9 +258,13 @@
 	attack_sound = 'sound/weapons/bladeslice.ogg'
 	faction = list("hostile")
 	mob_size = MOB_SIZE_TINY
-	is_flying_animal = TRUE
-	del_on_death = 1
 
 /mob/living/simple_animal/hostile/fairyswarm/Initialize()
 	. = ..()
 	AddComponent(/datum/component/swarming)
+
+//May allow for AOEs to work on it
+/mob/living/simple_animal/hostile/fairyswarm/spawned/death(gibbed)
+	animate(src, alpha = 0, time = 5 SECONDS)
+	QDEL_IN(src, 5 SECONDS)
+	..()
```

Comment:
```suggestion
/mob/living/simple_animal/hostile/fairyswarm/spawned/death(gibbed)
	animate(src, alpha = 0, time = (1 SECONDS))
	QDEL_IN(src, 1 SECONDS)
	..()
```
1. 5 seconds is too much for tiny fairies.
2. Add brackets in time argument, for some reason it will read "1 SECONDS" as "1" if you don't.